packages/synapse-react-client/src/components/InteractiveCopyIdsIcon.tsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react'
-import { IconButton, Tooltip } from '@mui/material'
+import { IconButton, IconButtonProps, Tooltip } from '@mui/material'
 import { TOOLTIP_DELAY_SHOW } from './SynapseTable/SynapseTableConstants'
 import IconSvg from './IconSvg/IconSvg'
 
-export type InteractiveCopyIdsIconProps = {
+export type InteractiveCopyIdsIconProps = Omit<IconButtonProps, 'onClick'> & {
   onCopy: () => void
 }
```

Comment:
This component can accept any `IconButtonProps` except for `onClick`, which will always just be `onCopy`

Reply:
Oh I see. Thanks for explaining. I guess "omit" here is for omitting "onClick" from IconButtonProps?